### PR TITLE
Adds lethal autoturrets to nuke operative uplink 

### PR DESCRIPTION
**Pull request by Victor239**
## About The Pull Request
Adds semi-automatic rounds (assault pod) and heavy laser autoturrets to nuke operative uplink. 

## Why It's Good For The Game
Nuke operatives can use these for suppressing fire or making parts of the station more hazardous in the same way they might otherwise use explosives, flamethrowers or assault cyborgs. 

## Changelog
:cl:
add: Semi-auto rounds and heavy laser autoturrets added to nuke operative uplink for 8 and 12 TC respectively.
/:cl:

### CODE REVIEW

File: code/game/machinery/syndicatebeacon.dm [hunk 1]
```diff
@@ -137,3 +137,15 @@
...
+	desc = "A label on it reads: <i>Warning: Activating this device will send a semi-auto turret to your location</i>."
+	droptype = /obj/machinery/porta_turret/syndicate/pod
+
+/obj/item/sbeacondrop/heavylaserturret
+	desc = "A label on it reads: <i>Warning: Activating this device will send a heavy laser turret to your location</i>."
+	droptype = /obj/machinery/porta_turret/syndicate/energy/heavy
+
+/obj/item/sbeacondrop/penetratorturret
+	desc = "A label on it reads: <i>Warning: Activating this device will send a penetrator turret to your location</i>."
+	droptype = /obj/machinery/porta_turret/syndicate/shuttle
```
Comment (Superyodeler) [inline code/game/machinery/syndicatebeacon.dm:151]: Needs a trailing newline, just add an empty line with no spaces or tabs to the end of the file.

**Comment by triplezeta** [comment]
> heavy laser turret


NO LASERS IN MY FUCKING UPLINK

**Comment by dinnerwoerror** [comment]
pshoom laysur

**Comment by Victor239** [comment]
Hate it to break it to ya but roboticists have the laser arm implant already.

**Comment by triplezeta** [comment]
I'm aware, that's getting axed by me Later™

**Pull request merged**